md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTypeAPI.java

```diff
@@ -69,7 +71,8 @@ RestWrapper get(
             restWrapper = new RestWrapper(processType, RestWrapper.OK);
             LOGGER.info("Record with ID:" + processTypeId + " selected from ProcessType by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)